.github/workflows/build-docker-image.yaml

```
@@ -155,31 +160,135 @@ jobs:
+          curl -Lo cosign https://github.com/sigstore/cosign/releases/download/${COSIGN_VERSION}/cosign-linux-amd64
+          chmod +x cosign
+          sudo mv cosign /usr/local/bin/cosign
+      - name: Setup GCP
+        id: gcp
+        uses: hoprnet/hopr-workflows/actions/setup-gcp@setup-gcp-v1
+        with:
+          google_credentials: ${{ secrets.gcp_service_account }}
+          login_artifact_registry: 'true'
+          install_sdk: 'true'
+      - name: Scan vulnerabilities with Trivy
+        run: |
```

actionlint (1.7.10)

property "docker" is not defined in object type {gcp: {conclusion: string; outcome: string; outputs: {string => string}}}

(expression)